benchmarks/image-processing/plugins/gatsby-source-remote-images/gatsby-node.js

```diff
@@ -0,0 +1,41 @@
+const fs = require(`fs`)
+const { createRemoteFileNode } = require(`gatsby-source-filesystem`)
+
+const NUM_IMAGES = parseInt(process.env.NUM_IMAGES || 100, 10)
```

Comment:
```suggestion
const NUM_IMAGES = Number(process.env.NUM_IMAGES || 100)
```

Reply:
We want an int not a Number (which could be a float)